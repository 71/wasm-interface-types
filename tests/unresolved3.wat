;; parse-fail

(module
  (@interface import "x" "y" (func (type $foo))))

(; CHECK-ALL:
failed to find type named `$foo`
     --> tests/unresolved3.wat:4:42
      |
    4 |   (@interface import "x" "y" (func (type $foo))))
      |                                          ^
;)
