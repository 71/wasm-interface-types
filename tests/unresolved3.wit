;; parse-fail

(module
  (@interface import "x" "y" (func (type $foo))))

(; CHECK-ALL:
  failed to find type named `$foo`
       --> tests/unresolved3.wit:4:42
        |
      4 |   (@interface import "x" "y" (func (type $foo))))
        |                                          ^
;)
