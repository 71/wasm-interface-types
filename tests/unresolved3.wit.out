failed to find type named `$foo`
     --> ../../tests/unresolved3.wit:4:42
      |
    4 |   (@interface import "x" "y" (func (type $foo))))
      |                                          ^
